org.fdroid.Repomaker.json

```diff
@@ -0,0 +1,260 @@
+                },
+                {
+                    "type": "file",
+                    "url": "https://files.pythonhosted.org/packages/35/d4/14e446a82bc9172d088ebd81c0b02c5ca8481bfeecb13c9ef07998f9249b/websocket_client-0.54.0.tar.gz",
+                    "sha256": "e51562c91ddb8148e791f0155fdb01325d99bb52c4cdbb291aee7a3563fd0849"
+                },
+                {
+                    "type": "file",
+                    "url": "https://files.pythonhosted.org/packages/d7/32/1e6be8c9ebc7d02fe74cb1a050008bc9d7e2eb9219f5d5e257648166e275/docker-py-1.9.0.tar.gz",
+                    "sha256": "6dc6b914a745786d97817bf35bfc1559834c08517c119f846acdfda9cc7f6d7e"
+                }
+            ]
+        },
+        {
+            "name": "openjdk",
+            "buildsystem": "simple",
+            "build-commands": [
+                "mkdir -p /app/java-9-openjdk/",
+                "cp -ra /usr/lib/sdk/openjdk9/jvm/openjdk-9.0.4/bin /usr/lib/sdk/openjdk9/jvm/openjdk-9.0.4/conf /usr/lib/sdk/openjdk9/jvm/openjdk-9.0.4/include /usr/lib/sdk/openjdk9/jvm/openjdk-9.0.4/lib /app/java-9-openjdk/"
+            ]
+        },
+        {
+            "name" : "repomaker",
+            "buildsystem" : "simple",
+            "build-commands" : [
+                "install -Dm644 org.fdroid.Repomaker.png /app/share/icons/hicolor/512x512/apps/org.fdroid.Repomaker.png",
+                "install -Dm644 org.fdroid.Repomaker.desktop /app/share/applications/org.fdroid.Repomaker.desktop",
+                "install -Dm644 org.fdroid.Repomaker.appdata.xml /app/share/metainfo/org.fdroid.Repomaker.appdata.xml",
+                "pip3 install --no-index --find-links=\"file://${PWD}\" --prefix=${FLATPAK_DEST} repomaker",
+                "pip3 install --no-deps --no-index --find-links=\"file://${PWD}\" --prefix=${FLATPAK_DEST} pywebview"
+            ],
+            "sources" : [
+                {
+                    "type" : "file",
+                    "path" : "app/share/applications/org.fdroid.Repomaker.desktop"
+                },
+                {
+                    "type" : "file",
+                    "path" : "app/share/metainfo/org.fdroid.Repomaker.appdata.xml"
+                },
+                {
+                    "type" : "file",
+                    "path" : "app/share/icons/hicolor/512x512/apps/org.fdroid.Repomaker.png"
+                },
+                {
+                    "type": "file",
+                    "url": "https://files.pythonhosted.org/packages/a3/a8/32af11b7893d4d4239df5496a545dc870a7c41b99c5418199a384768920a/pywebview-2.2.1.tar.gz",
+                    "sha256": "b680dcc517a8b742b1abec6db626d3795b7a5b589af36c0bc022febc3285904b"
+                },
+                {
+                    "type" : "file",
+                    "url" : "https://media.dorfbrunnen.eu/repomaker-0.0.5.tar.gz",
+                    "sha256": "4bc4376d8c38eb52607cb53d8e6d4076138f3b09eded6091963448321378a79f"
```

Comment:
See [this MR at upstream's repo](https://gitlab.com/fdroid/fdroid-repomaker-flatpak/merge_requests/19) on why a custom package built by me is used here.

Reply:
This is planned to be switched back to official PyPi versions. We can also do this before inclusion into Flathub, if requested.